lib/DirectAdmin_API.php

```diff
@@ -47,14 +47,32 @@ public function __construct($host = null, $login = null, $pass = null, $secure =
 			$this->_setupLocalConnection();
 	}
 	
-	public function getDomainsMxRecords($domain){
+	public function getDomainsMxRecords($domain, $username){
 		$sock = $this->_getSocket();
 		$sock->query('/CMD_API_DNS_ADMIN?domain='.$domain.'&urlencoded=yes');
 		$result = $sock->fetch_body();
 		
 		if (!$result){
 			throw new Exception('Unable to get DNS zone of domain ' . $domain);
 		}
+
+		$isRemote = FALSE;
+		if (strlen($username) != 0) {
+			$sock->query('/CMD_API_DNS_MX?domain='.$domain.'&as_user='.$username.'&urlencoded=yes');
+			$mx = $sock->fetch_body();
```

Comment:
__[phpmd]__
* Avoid variables with short names like $mx. Configured minimum length is 3.

<!-- Comment by [SideCI](https://sideci.com) -->